explore/src/main/scala/explore/targeteditor/SiderealTargetEditor.scala

```diff
@@ -273,6 +273,7 @@ object SiderealTargetEditor {
                   // SearchForm doesn't edit the name directly. It will set it atomically, together
                   // with coords & magnitudes from the catalog search, so that all 3 fields are
                   // a single undo/redo operation.
+                  allView,
```

Comment:
maybe `allView` can replace the use of `targetView` as well, but we can take a look later